.github/workflows/checkCourseName.yml

```diff
@@ -0,0 +1,52 @@
+name: Auto review course name 
+
+on: 
+  pull_request_target:
+    branches: [ main ]
+
+  workflow_dispatch: 
+jobs: 
+  auto_review: 
+    runs-on: ubuntu-latest 
+    env:
+      PR_NUMBER: ${{github.event.number}}
+
+    steps: 
+      - name: Checkout playbooks
+        uses: actions/checkout@v2 
+        with: 
+          repository: devonfw-tutorials/tutorials 
+          path: playbooks 
+
+      - id: changedfiles
+        uses: jitterbit/get-changed-files@v1
+
+      - name: Check name 
+        id: check_name
+        run: | 
+          cd playbooks
+          playbook_list=()
+          for dir in */; do 
+            playbook_list+=("${dir::-1}"); 
+          done
+          message=""
+          for course_file in ./*; do
+            if [[ $course_file = *"pathway"* ]]; then
+              coursefile=${course_file/#.\/}
+              course_name=${coursefile/%-pathway.json}
+              if [[ " ${playbook_list[@]} " =~ " ${course_name} " ]]; then
+                message="${message} ${course_name} exist already\n"
+              fi
+            fi
+          done
+          if [[ message != "" ]]; then
+            message="${message} You just tried to name a course that is already a playbook.\n Please change the name. \n"
```

Comment:
I think you have to remove the line breaks in the message. I also wanted to add some in the workflow for auto reviewing. But that resulted in the text not being displayed after the first line break.